lib/helper/WebDriver.js

```diff
@@ -1581,6 +1581,41 @@ class WebDriver extends Helper {
     await browser.buttonUp(0);
   }
 
+  /**
+   * Get all Window Handles.
+   * Useful for referencing a specific handle when calling `I.switchToWindow(handle)`
+   *
+   * ```js
+   * const handles = await I.getWindowHandles();
+   * ```
+   */
+  async getWindowHandles() {
+    return this.browser.getWindowHandles();
+  }
+
+  /**
+   * Get the current Window Handle.
+   * Useful for referencing it when calling `I.switchToWindow(handle)`
+   *
+   * ```js
+   * const handle = await I.getWindowHandle();
+   * ```
+   */
+  async getWindowHandle() {
+    return this.browser.getWindowHandle();
+  }
+
+  /**
+   * Switch to the window with a specified handle.
+   *
+   * ```js
+   * await I.switchToWindow( handle );
```

Comment:
example should include grab* actions listed above.

Reply:
so to know where to get those window handles

Reply:
Also making these adjustments. Thanks for reviewing.